games/applications/grovers.py

```diff
@@ -0,0 +1,263 @@
+# Copyright 2022 SRI-International organization
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cff-version: 1.2.0
+# "This code below was adapted from the code created by the SRI-International organization."
+# The names listed below are the orginal authors of the code
+# authors:
+#   - family-names: Lubinski
+#     given-names: Thomas
+#   - family-names: Johri
+#     given-names: Sonika
+#   - family-names: Varosy
+#     given-names: Paul
+#   - family-names: Coleman
+#     given-names: Jeremiah
+#   - family-names: Zhao
+#     given-names: Luning
+#   - family-names: Necaise
+#     given-names: Jason
+#   - family-names: Baldwin
+#     given-names: Charles
+#   - family-names: Mayer
+#     given-names: Karl
+#   - family-names: Proctor
+#     given-names: Timothy
+# title: "Application-Oriented Performance Benchmarks for Quantum Computing"
+# date-released: 2021-10-7
+# url: "https://arxiv.org/abs/2110.03137"
+# github repository:
+# https://github.com/SRI-International/q_c-App-Oriented-Benchmarks/blob/master/grovers/qiskit/grovers_benchmark.py
```

Comment:
This would preferably be a permalink so that it refers to the exact version you copied/forked.